exercises/perfect-numbers/src/perfect_numbers.rc

```diff
@@ -0,0 +1 @@
+# Please implement your solution to perfect-numbers in this file
```

Comment:
Should be `exercises/perfect-numbers/src/perfect_numbers.cr`.

Reply:
Oh, yes! Sorry about that! Will fix tonight!